include/jemalloc/internal/tcache_inlines.h

```diff
@@ -33,29 +33,44 @@ tcache_enabled_set(tsd_t *tsd, bool enabled) {
 }
 
 static inline unsigned
-tcache_nhbins_get(tcache_t *tcache) {
+tcache_nbins_get_slow(tcache_slow_t *tcache_slow) {
```

Comment:
The two `_slow` postfix can be a bit confusing. Usually when we name it that way, it implies some non-trivial stuff happening and should be avoided as much as possible. Here's it's still plenty fast, and we honestly access tcache_slow pretty often.

Let's only provide tcache_nbins / max_get(), which take tcache_slow as input. Most call sites should have tcache_slow in the context, and if not, passing tcache->tcache_slow explicitly is a good thing.